### DISCUSSION TITLE
Can Flow.Launcher set Double ctrl as a wake-up shortcut key?

### DISCUSSION THREAD

**Discussion by Ahernz** [Flow-Launcher/Flow.Launcher#2046] [Q&A] [answered]
Can Flow.Launcher set Double ctrl as a wake-up shortcut key?
Just like Listary.Press ctrl twice to display the input box.

**Comment by jjw24** [comment]
This is something we would like to implement in the future but currently not possible with the hotkey library flow is using.

**Reply by Ahernz** [reply]
I have another need，
When I use custom hotkeys for queries, 
how can I set "folder" to be fixed and not editable?
![2023-04-13_22-40-47](https://user-images.githubusercontent.com/76786517/231795936-bd9382e9-9707-4343-8bbb-9885b95ab814.jpg)
![2023-04-13_22-39-50](https://user-images.githubusercontent.com/76786517/231795940-025d94c5-3071-4371-a4db-0c34df012302.jpg)

**Reply by Ahernz** [reply]
I hope this feature can be implemented as soon as possible because it is a useful feature.
Thank you for developing such a great software, and I have donated it. 
Please let me buy you a cup of coffee.

**Reply by jjw24** [reply]
> I hope this feature can be implemented as soon as possible because it is a useful feature. Thank you for developing such a great software, and I have donated it. Please let me buy you a cup of coffee.

Thank you @Ahernz for the donation :) Good to hear you are enjoying flow.

**Reply by jjw24** [reply]
>When I use custom hotkeys for queries,
how can I set "folder" to be fixed and not editable?

What do you mean fixed and not editable sorry?

**Reply by Ahernz** [reply]
I'm sorry, I don't know English, so I can't express the correct meaning. 
I don't know if you understand Chinese. I want to describe my thoughts in Chinese.

我设置了快捷键Ctrl+~唤起Flow.Launcher，并且输入指令“Foleer:”，
当我在指令“Foleer:”后面输入文字的时候，
他可以让我在进行搜索的时候，只搜索文件夹而不搜索文件。
我是想问能否在用快捷键查询的时候，把提前输入的“Foleer:”作为一个固定项，
因为有时候我在修改文字的时候，会误删掉“Foleer:”。

**Reply by VictoriousRaptor** [reply, marked as answer]
> > When I use custom hotkeys for queries,
> > how can I set "folder" to be fixed and not editable?
> 
> What do you mean fixed and not editable sorry?

What he requests is to make the query uneditable when using custom query hotkey cuz sometimes he accidentally delete his "Folder:" custom query.

Personally I think it's possible but may require some annoying checks. At least it should be configurable for each custom hotkey. An alternative solution for this is just to press the hotkey again to input the custom query.

这在技术上可行但可能会引入一些麻烦。当你误删的时候可以再按一次快捷键来重新触发自定义查询。

**Reply by Ahernz** [reply]
Thank you, I think the second solution solved my problem.
